plugins/volumeFilter_test.go

```diff
@@ -152,6 +152,215 @@ func TestMakeFilterVolume(t *testing.T) {
 	}
 }
 
+// volumeFilterFnTestCase is the input that will be reused across all tests of type TestVolumeFilterFn*
+type volumeFilterFnTestCase struct {
+	name         string
+	cap          float64
+	otb          float64
+	tbbBase      float64
+	tbbQuote     float64
```

Comment:
I think we may be able to remove `tbbQuote` and rename `tbbBase` to just `tbb`.
tbb values are analogous to their `cap` and `otb` counterparts.
since we plan to reuse this struct for all tests include base/quote we have not named `otb`, `tbb`, and `cap` with the base/quote suffix since it will be used differently in different tests

Tip: a good way to find extra / unused variables is when their value doesn't change across test cases. In this case `tbbQuote` is always 0 in the test cases below (which makes sense because we are constraining on the base side).

Reply:
Good point. I had `tbbQuote` since we need to accumulate the values on the tbb. But this can be easily changed by replacing the input with `pointy.Float64(0)`, rather than having this value on every struct.